docs/source/building_applications/rag.md

```diff
@@ -178,3 +179,31 @@ for vector_db_id in client.vector_dbs.list():
     print(f"Unregistering vector database: {vector_db_id.identifier}")
     client.vector_dbs.unregister(vector_db_id=vector_db_id.identifier)
 ```
+
+### Appendix
+
+#### More RAGDocument Examples
+```python
+from llama_stack_client import RAGDocument
+RAGDocument(
+    document_id="num-0",
+    content={"uri": "file://path/to/file"}
+)
+RAGDocument(
+    document_id="num-1",
+    content="plain text"
+)
+RAGDocument(
+    document_id="num-2",
+    content={"type": "text", "text": "plain text input"}  # for inputs that should be treated as text explicitly
+)
+RAGDocument(
+    document_id="num-3",
+    content={"type": "image", "image": {"url": {"uri": "https://mywebsite.com/image.jpg"}}}
+)
+RAGDocuemnt(
+    document_id="num-4"
+    content={"type": "image", "image": {"data": "b64 encoded image"}}
```

Comment:
It would probably be useful to show how to base64 encode an image in a comment (or even the documentation)